### PR TITLE
allowed to use dropout in vgg

### LINKED ISSUE #16550: Possibility to use dropout in vgg16/19
If you open a GitHub issue, here is our policy:

It must be a bug, a feature request, or a significant problem with the documentation (for small docs fixes please send a PR instead).
The form below must be filled out.

**Here's why we have that policy:**.

Keras developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

**System information**.

TensorFlow version (you are using): 2.8
Are you willing to contribute it (Yes/No) : Yes

**Describe the feature and the current behavior/state**.

I would like to be able to use dropout on the first two fully connected layers of the VGG implementations in keras applications, as specified [in the original paper](https://arxiv.org/abs/1409.1556) (and [done in PyTorch](https://github.com/pytorch/vision/blob/main/torchvision/models/vgg.py#L46)).

> dropout regularisation for the first two fully-connected layers (dropout ratio set to 0.5)

**Will this change the current api? How?**

Yes: this will mean adding a `dropout` kwarg to the VGG implementations, that will default to `0.0` to remain a non-breaking change.

**Who will benefit from this feature?**

Users willing to reproduce the original results of the VGG paper.

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**

- Do you want to contribute a PR? (yes/no): yes
- If yes, please read [this page](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md) for instructions
- Briefly describe your candidate solution(if contributing): just adding dropout layers.

### PR DESCRIPTION
**Pull request by zaccharieramzi**
This closes #16550

### CODE REVIEW

**Comment by fchollet** [comment]
Thanks for the PR. We're not modifying the API of Keras Applications to add regularization. If you need to use regularization with this model, please simply copy and modify to code for your own needs.

**Comment by zaccharieramzi** [comment]
Makes sense!
